tests/test_read_topas_files.py

```diff
@@ -0,0 +1,118 @@
+import os
+
+import numpy as np
+from pymchelper.estimator import Estimator
+from pymchelper.readers.topas import TopasReaderFactory
+
+
+def test_read():
+    """Test if Topas output file is read correctly"""
+    file_path = os.path.join("tests", "res", "topas", "minimal", "fluence_bp_protons_xy.csv")
```

Comment:
move that to a fixture, see for example https://github.com/DataMedSci/pymchelper/blob/2a7c6a560284f42e37f0aa09ed0fe5327f83eb72/tests/test_mcpl.py#L13